src/main/java/edition/academy/seventh/service/BookService.java

```diff
@@ -21,20 +21,20 @@ public BookService(BookRepository bookRepository) {
   }
 
   /**
-   * Adds given books to database {@link edition.academy.seventh.database.model.Book}.
+   * Adds given books to database {@link DtoBook}.
    *
-   * @param books list of books to be added
+   * @param dtBooks {@code List<DtoBook>} to be added
    */
-  public void addBooksToDatabase(List<Book> books) {
-    bookRepository.addBooksToDatabase(books);
+  public void addBooksToDatabase(List<DtoBook> dtBooks) {
+    bookRepository.addBooksToDatabase(dtBooks);
   }
 
   /**
-   * Retrieves book entities from database and packs them into the List
+   * Retrieves book entities from database and packs them into the {@code List<DtoBook>}.
    *
-   * @return List of books.
+   * @return {@code List<DtoBook>}.
```

Comment:
to widać z deklaracji metody, tu powinien się znaleźć opis tego co jest zwracane i tu nie chodzi o typ danych tylko np co konkretnie zawiera ta lista -> czyli książki zapisane w bazie obecnie